Makefile

```diff
@@ -45,7 +46,7 @@ profile: examples/filter-simple/main-debug.wasm
 
 .PHONY: bench-plugin
 bench-plugin:
-	@(cd internal/e2e; go test -run='^$$' -bench '^BenchmarkPlugin.*$$' . -count=6)
+	@(cd internal/e2e; go test -timeout 30m -run='^$$' -bench '^BenchmarkPlugin.*$$' . -count=6)
```

Comment:
added a longer timeout as I made another bench dimension, but actually I might just refactor that...

Reply:
removed the "wat" benchmark as it isn't important the difference between noop wat and tinygo